src/breadcrumb/breadcrumb.component.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { Component } from "@angular/core";
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { FormsModule } from "@angular/forms";
+import { By } from "@angular/platform-browser";
+import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from "@angular/platform-browser-dynamic/testing";
+
+import { BreadcrumbComponent } from "./breadcrumb.component";
+import { BreadcrumbItemComponent } from "./breadcrumb-item.component";
+import { BreadcrumbItem } from "./breadcrumb-item.interface";
+import { OverflowMenu } from "../dialog/overflow-menu/overflow-menu.component";
+
+@Component({
+	selector: "test-breadcrumb",
+	template: `
+	<ibm-breadcrumb [noTrailingSlash]="_noTrailingSlash">
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 1
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 2
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 3
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 4
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 5
+		</ibm-breadcrumb-item>
+	</ibm-breadcrumb>`,
+	entryComponents: [BreadcrumbComponent]
+})
+class TestBreadcrumbComponent {
+	_noTrailingSlash = true;
+
+	set noTrailingSlash(val: boolean) {
+		this._noTrailingSlash = val;
+	}
+}
+
+@Component({
+	selector: "test-breadcrumb",
+	template: `
+	<ibm-breadcrumb
+		[noTrailingSlash]="_noTrailingSlash"
+		[overflowThreshold]="_overflowThreshold"
+		[breadcrumbItems]="_breadcrumbItems">
+	</ibm-breadcrumb>`,
+	entryComponents: [BreadcrumbComponent]
+})
+class TestBreadcrumbModelComponent {
+	_noTrailingSlash = true;
+	_overflowThreshold = 4;
+	_breadcrumbItems = [];
+
+	set noTrailingSlash(val: boolean) {
```

Comment:
Why setters for these?

Reply:
So we can test inputs on the breadcrumb components

Reply:
I'd either expose the whole breadcrumb component or swap out for straight inputs ... as far as I can see there really isn't a requirement to have setters like that ... I'm definitely still learning when it comes to Angular testing though 👍 so if there's something I'm missing let me know

Reply:
@cal-smith 
AFAIK this is the recommended pattern for testing @Input() properties, see https://medium.com/@AikoPath/testing-angular-components-with-input-3bd6c07cfaf6

Reply:
It's certainly _a_ pattern for testing Inputs, but I don't see anything obvious in that blogpost that _requires_ setters ... In the post for example they just use "old school setters" (`setMethod`) and later abandon them in favor of directly accessing the component. 

If it was wrapping up some additional logic it would be 💯 but all it's doing is `this.prop = value`, so I don't really see the purpose.